### ISSUE TITLE
`ValidatorPluginManager` must not require a template type

### ISSUE THREAD

**Issue by boesing** [laminas/laminas-validator#209]
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 2.39.0

#### Summary

The `ValidatorPluginManager` has an `@template` annotation. Plugin manager implementations must not have templates, they do just pass the template type to the parent plugin manager.

#### Current behavior

```
Argument 1 of Laminas\Validator\ValidatorChain::attach cannot be (InstanceType:Laminas\Validator\ValidatorPluginManager as Laminas\Validator\ValidatorInterface)|mixed, expecting Laminas\Validator\ValidatorInterface (see https://psalm.dev/030)
```

#### How to reproduce

Fetch any validator via plugin manager and verify the type using `@psalm-trace`

#### Expected behavior

Return type should be `ValidatorInterface` or concrete implementation of a plugin if `class-string` provided.

**Comment by boesing** [comment]
Refs: https://github.com/laminas/laminas-serializer/blob/e8112262e131dffeeb9a0a6ddb69cdb4bf2c3da5/src/AdapterPluginManager.php#L21

**Comment by mimmi20** [comment]
Is this already fixed?

**Comment by gsteel** [comment]
Should be: https://github.com/laminas/laminas-validator/commit/70692fa0c551961bf961de6236c50c60dc3b0353

**Comment by gsteel** [comment]
Fixed in #182
